app/src/org/commcare/tasks/ProcessAndSendTask.java

```diff
@@ -272,6 +272,8 @@ private void sendForms(FormRecord[] records) {
                             }
                             results[i] = FormUploadUtil.sendInstance(i, folder, new SecretKeySpec(record.getAesKey(), "AES"), url, this, mUser);
                             if (results[i] == FormUploadResult.FULL_SUCCESS) {
+                                Logger.log(AndroidLogger.TYPE_FORM_SUBMISSION,

```

Comment:
Flagging that this is going to be _really_ verbose overall. Might be unavoidable, but seems a bit unfortunate.